<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstallVoicesDialog</class>
 <widget class="QDialog" name="InstallVoicesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Install/Remove Festival voices</string>
  </property>
  <property name="windowIcon">
   <iconset resource="omilo.qrc">
    <normaloff>:/images/audio-input.png</normaloff>:/images/audio-input.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="AwbLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Awb Cmu Festival</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="awbInstallButton">
         <property name="text">
          <string>Install</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="awbRemoveButton">
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="BdlLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Bdl Cmu Festival</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bdlInstallButton">
         <property name="text">
          <string>Install</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bdlRemoveButton">
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="ClbLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Clb Cmu Festival</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clbInstallButton">
         <property name="text">
          <string>Install</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clbRemoveButton">
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="JmkLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Jmk Cmu Festival</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="jmkInstallButton">
         <property name="text">
          <string>Install</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="jmkRemoveButton">
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLineEdit" name="KspLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Ksp Cmu Festival</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="kspInstallButton">
         <property name="text">
          <string>Install</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="kspRemoveButton">
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLineEdit" name="RmsLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Rms Cmu Festival</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rmsInstallButton">
         <property name="text">
          <string>Install</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rmsRemoveButton">
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLineEdit" name="SltLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Slt Cmu Festival</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sltInstallButton">
         <property name="text">
          <string>Install</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sltRemoveButton">
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>Close</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>AwbLineEdit</tabstop>
  <tabstop>awbInstallButton</tabstop>
  <tabstop>awbRemoveButton</tabstop>
  <tabstop>BdlLineEdit</tabstop>
  <tabstop>bdlInstallButton</tabstop>
  <tabstop>bdlRemoveButton</tabstop>
  <tabstop>ClbLineEdit</tabstop>
  <tabstop>clbInstallButton</tabstop>
  <tabstop>clbRemoveButton</tabstop>
  <tabstop>JmkLineEdit</tabstop>
  <tabstop>jmkInstallButton</tabstop>
  <tabstop>jmkRemoveButton</tabstop>
  <tabstop>KspLineEdit</tabstop>
  <tabstop>kspInstallButton</tabstop>
  <tabstop>kspRemoveButton</tabstop>
  <tabstop>RmsLineEdit</tabstop>
  <tabstop>rmsInstallButton</tabstop>
  <tabstop>rmsRemoveButton</tabstop>
  <tabstop>SltLineEdit</tabstop>
  <tabstop>sltInstallButton</tabstop>
  <tabstop>sltRemoveButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources>
  <include location="omilo.qrc"/>
 </resources>
 <connections/>
</ui>
